io.gitlab.caveman250.headlines.json

```diff
@@ -0,0 +1,283 @@
+{
+  "app-id": "io.gitlab.caveman250.headlines",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "41",
+  "sdk": "org.gnome.Sdk",
+  "command": "headlines",
+  "finish-args": [
+    "--share=network",
```

Comment:
you still need `--share=ipc` for x11 (even as fallback)

Reply:
fixed.